### PR TITLE
Prepare for 11.11.0 release

### PR DESCRIPTION
**Pull request by scpeters**
Comparison to 11.10.2: https://github.com/osrf/gazebo/compare/gazebo11_11.10.2...gazebo11

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 ## Gazebo 11
 
+## Gazebo 11.11.0 (2022-05-25)
+
+1. Convert `OGRE_RESOURCE_PATH` with `TO_CMAKE_PATH`
+    * [Pull request #3165](https://github.com/ignitionrobotics/gazebo/pull/3165)
```
Comment (jennuine) [inline Changelog.md]: This and the links below need to be updated:

```suggestion
    * [Pull request #3165](https://github.com/osrf/gazebo/pull/3165)
```

**Review by jennuine — APPROVED** [review]
LGTM, one minor comment

File: Changelog.md (hunk 1 above)
Comment (scpeters) [reply Changelog.md]: I do this every time...

Comment (scpeters) [reply Changelog.md]: https://github.com/osrf/gazebo/pull/3221/commits/395b260ec7830d29a4db9d45b6fd8d43bee18d6c

**Comment by scpeters** [comment]
I've been reviewing some test failures and noticed some regressions introduced by recent pull requests. I'll submit a quick fix for some of them

**Comment by scpeters** [comment]
> I've been reviewing some test failures and noticed some regressions introduced by recent pull requests. I'll submit a quick fix for some of them

holding for https://github.com/osrf/gazebo/pull/3223

**Comment by scpeters** [comment]
this just changes the Changelog and version number, so I'm not going to wait for CI again. Merging...

**Pull request merged**